### PR TITLE
Hotfix diplo

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description
Adds a mechanism to stream by mp3 instead of HLS and exposes that option via optimizely.

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/hotfix-diplo

File: src/services/audius-api-client/AudiusAPIClient.ts [hunk 1]
```diff
@@ -915,6 +915,17 @@ class AudiusAPIClient {
     console.debug('APIClient: Initialized')
   }
 
+  makeUrl = async (
```
Comment (jowlee) [inline src/services/audius-api-client/AudiusAPIClient.ts:918]: don't think this needs to be async

File: src/audio/AudioStream.ts [hunk 2]
```diff
@@ -184,79 +184,91 @@ class AudioStream {
     onEnd: () => void,
     prefetchedSegments = [],
     gateways = [],
-    info = { title: '', artist: '' }
+    info = { id: '', title: '', artist: '' },
```
Comment (jowlee) [inline src/audio/AudioStream.ts:187]: You added id here, but I don't think it's used.

**Review by jowlee — APPROVED** [review]
Worked w/ diplo's track locally against prod for me.

**Pull request closed without merging**